docs/topics/windows-provisioning-scripts.md

```diff
@@ -0,0 +1,29 @@
+# Windows Provisioning Scripts
+
+## Overview
+
+Due to requirements around signing executable code on Windows nodes some powershell scripts are are delivered via a .zip file during node provisioning.
```

Comment:
"are are"

Also, can you be more specific about the requirements? Customers might get confused and not know if this applies to them

Reply:
I don't follow. What part do you think might lead to confusion and do you have any suggestions on how to make things more clear?

Reply:
I think just include a sentence or two on why we are signing the scripts in the first place.

Reply:
Added